simulator/src/main/java/com/hazelcast/simulator/coordinator/PerformanceStateContainer.java

```diff
@@ -86,9 +86,9 @@ public String formatPerformanceNumbers(String testCaseId) {
             return "";
         }
         String latencyUnit = "µs";
-        long avgLatencyValue = round(performanceState.getIntervalAvgLatency());
-        long percentileLatencyValue = performanceState.getIntervalPercentileLatency();
-        long maxLatencyValue = performanceState.getIntervalMaxLatency();
+        long avgLatencyValue = round(performanceState.getIntervalAvgLatency() / MICROSECONDS.toNanos(1));
```

Comment:
I'll fix it by renaming the fields in the performancestate to indicate their unit.